cypress/utils/index.d.ts

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+declare namespace Cypress {
+  interface Chainable<Subject> {
+    /**
+     * Call a function when an element with a test id cannot be found
+     * @example
+     * cy.whenTestIdNotFound(['query', 'puery'], () => {...})
+     */
+    whenTestIdNotFound<S = any>(
```

Comment:
@d-rowe @ananzh so it seems we are already migrating from that package in here too right?

Reply:
also do people call this?